### PR TITLE
#13655 - Assets/Collection - Tests finished - Just 2 errors in Phalcon

### PR DESCRIPTION
**Pull request by jenovateurs**
Hi team!
I have a new delivery of test. 
Assets/Collection is done.
But I have 3 problems : 
- GitHub display a message : " Can’t automatically merge. Don’t worry, you can still create the pull request. " - I don't know how to solved it.
- The test Assets/Collection/CurrentCest trigger an error because the variable "position" is not initialize in Phalcon (Error : phalcon/Assets/Collection.zep on line 261)
- I have the same problem with Asset/Collection/ValidCest, I can't valid my CSS file because "position" is not initialize.

Thanks for your help ;-)

* Type: code quality
* Link to issue: #13655

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Thanks

### CODE REVIEW

**Comment by Jeckerson** [comment]
> " Can’t automatically merge. Don’t worry, you can still create the pull request. " - I don't know how to solved it.

You need to rebase your branch with latest commits from `4.0.x`

**Comment by Jeckerson** [comment]
> The test Assets/Collection/CurrentCest trigger an error because the variable "position" is not initialize in Phalcon (Error : phalcon/Assets/Collection.zep on line 261)

You can try to set default value:
`protected position = 0 { get };`

https://github.com/phalcon/cphalcon/blob/master/phalcon/Assets/Collection.zep#L74

**Comment by jenovateurs** [comment]
@Jeckerson Thanks for your help ;-)

File: phalcon/Assets/Collection.zep [hunk 1]
```diff
@@ -71,7 +71,7 @@ class Collection implements Countable, Iterator
     /**
      * @var int
      */
-    protected position { get };
+    protected position = 0 { get };
```
Comment (ruudboon) [inline phalcon/Assets/Collection.zep]: Looks like you solved a bug as well. Can you add an entry to the change log?

**Review by ruudboon — CHANGES_REQUESTED** [review]

File: phalcon/Assets/Collection.zep (hunk 1 above)
Comment (jenovateurs) [reply phalcon/Assets/Collection.zep]: It seems :-), @Jeckerson helps me to solve this problem. And ok about the change log.

**Review by ruudboon — APPROVED** [review]

**Pull request merged**